getting_started/README.md

```diff
@@ -2,12 +2,10 @@
 
 Roc is a language for making delightful software. It does not have an 0.1 release yet, and we
 certainly don't recommend using it in production in its current state! However, it can be fun to
-play around with as long as you have a high tolerance for missing features and compiler bugs. :)
+play around with as long as you have a tolerance for missing features and compiler bugs. :)
 
 The [tutorial](https://roc-lang.org/tutorial) is the best place to learn about how to use the language - it assumes no prior knowledge of Roc or similar languages. (If you already know [Elm](https://elm-lang.org/), then [Roc for Elm Programmers](https://github.com/roc-lang/roc/blob/main/roc-for-elm-programmers.md) may be of interest.)
 
-There's also a folder of [examples](https://github.com/roc-lang/roc/tree/main/examples) - the [CLI form example](https://github.com/roc-lang/roc/tree/main/examples/cli/form.roc) in particular is a reasonable starting point to build on.
-
```

Comment:
We mention examples later on in the document, and I think we no longer recommend CLI form to start with.